### ISSUE TITLE
Improve the workflow for adding properties

### ISSUE THREAD

**Issue by tristanlatr** [LesFruitsDefendus/saskatoon-ng#71]
When a tree owner confirm it's participation to the season, they fills the `create_pending` form.
This form can either lead to creating the whole new property or to merge the information with already existing property.
This process is very frustrating for the core members, since we need to check manually the address of the property. 
We need some semi-automated workflow to determine if it's a duplicate or a very new property. 

The current process reads as follow:

```
Process to handle pending properties :
- Open two different windows of Saskatoon running aside another, this will save you lot of time.
- Filter properties to show only pending validation
- For each pending property, do the following :
- Check the address your second Saskatoon window open on properties tab to see if the property exists.
- If the property exists, do the following :
- Click edit property on both new created and original property pages
- Update all relevant infos from the newly created to the original property by comparing fields
- Check if a “Auth user” and “Person” record is assigned to the property, create it if need be
- Uncheck “pending validation” and “is active” from newly created property
- Input “DELETE” as the street of the new property
- Save original property
- Save new property
- If the property doesn’t exist, do the following :
- With the second Saskatoon tab, from admin panel, create a “Auth user" - leaving password field empty - and add a “Person” record with all infos available for the tree owner. 
- Enter email address in comments box (This part might change)
- Click on edit property
- Assign the newly created Person record to be the owner of the property
- Uncheck “pending validation”
- Add any other relevant infos to property record
- Save new property
```

As you can see, it's really painful. This has to change!

**Comment by tiagovaz** [comment]
As we discussed in the last meeting, we could think of generating a one-time link for the property update by the owner. Maybe using the following: https://simpleisbetterthancomplex.com/tutorial/2016/08/24/how-to-create-one-time-link.html

**Comment by qusvv** [comment]
I'm starting to look at this:
- issue #83 is closely related, although I now understand better that a property has a `pending_email` field so more thoughts on this are needed
- as it is now, `pending_contact_name/phone/email` don't appear in the property details when its status is `pending`
- I am then confused as to how one would see the pending information since even in the admin panel `PropertyAdmin` uses the `PropertyForm` and not `PublicPropertyForm`

I see this in the old-generation (on `develop` branch but not on `master`):
```
                   {% if user.is_superuser %}
                    <strong>{% trans "Name" %} : </strong>{{ property.pending_contact_name }}<br>
                    <strong>{% trans "Phone" %} : </strong>{{ property.pending_contact_phone }}<br>
                    <strong>{% trans "Email" %} : </strong>{{ property.pending_contact_email }}<br><br>
```

which explains why a superuser can have access to ~pending_contact_*` information
e.g. https://saskatoon.lesfruitsdefendus.org/harvest/property/517

@tiagovaz can you confirm that the prod server is currently running the `develop` branch and not `master` of saskatoon?

**Comment by tiagovaz** [comment]
Yes, current prod is running develop branch. I can't recall why but there was something blocking in the way that I couldn't give priority. I think we could leave like that since it's not going to be used anymore.

**Comment by qusvv** [comment]
I think the first thing that should be done is to implement a `Auth.user`+`Person` check/creation on save(), similar to the one implemented in `RequestForm`,  by  comparing `the pending_contact_email` to the auth user email. This will at least save core member to task:
`Check if a “Auth user” and “Person” record is assigned to the property, create it if need be`

This way duplicate properties would be associated with the same person and will be easier to compare.
